left_leg_motor_0:
  id: 0
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
left_leg_motor_1:
  id: 1
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
left_leg_motor_2:
  id: 2
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
left_leg_motor_3:
  id: 3
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
left_leg_motor_4:
  id: 4
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
left_leg_motor_5:
  id: 5
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
right_leg_motor_0:
  id: 6
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
right_leg_motor_1:
  id: 7
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
right_leg_motor_2:
  id: 8
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
right_leg_motor_3:
  id: 9
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
right_leg_motor_4:
  id: 10
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
right_leg_motor_5:
  id: 11
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
left_arm_motor_0:
  id: 12
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
left_arm_motor_1:
  id: 13
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
right_arm_motor_0:
  id: 14
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
right_arm_motor_1:
  id: 15
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
head_motor_0:
  id: 16
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
head_motor_1:
  id: 17
  type: "ax12"
  angle_zero: 150
  angle_max: 300
  angle_min: 0
